package.json

```diff
@@ -32,11 +43,21 @@
     "@stylistic/eslint-plugin-js": "^1.4.1",
     "eslint": "^8.54.0",
     "eslint-config-google": "^0.14.0",
-    "tree-sitter-cli": "^0.20.8"
+    "tree-sitter-cli": "^0.22.0",
+    "prebuildify": "^6.0.0"
   },
   "dependencies": {
-    "nan": "^2.18.0",
-    "tree-sitter-html": "^0.20.0"
+    "tree-sitter-html": "^0.20.1",
+    "node-addon-api": "^7.1.0",
+    "node-gyp-build": "^4.8.0"
+  },
+  "peerDependencies": {
+    "tree-sitter": "^0.21.0"
+  },
+  "peerDependenciesMeta": {
+    "tree_sitter": {
+      "optional": true
+    }
   },
   "tree-sitter": {
```

Comment:
This is actually incorrect (and thus breaks `:TSInstallFromGrammar`); it needs to be an array: `"tree-sitter": [ {...} ]`.

Reply:
Ah, thanks for catching this one